Source/View Controllers/TCCProfileViewController.swift

```diff
@@ -171,6 +202,7 @@ class TCCProfileViewController: NSViewController {
             switch tccProfileResult {
```

Comment:
Since all of this code relies on `self` not being weak, a `guard` statement might be appropriate.

```suggestion
            guard strongSelf = self else { return }
            switch tccProfileResult {
```

Then you'd also change the references to `self?` below to `strongSelf`.